perfkitbenchmarker/data/cassandra/create-ycsb-table.cql.j2

```diff
@@ -12,10 +12,17 @@
  # See the License for the specific language governing permissions and
  # limitations under the License. #}
 {# Create a table for YCSB benchmark. #}
-create keyspace {{ keyspace }}
-  with placement_strategy = 'SimpleStrategy'
-  and strategy_options = {replication_factor: {{ replication_factor }} };
-
-use {{ keyspace }};
-
-create column family {{ column_family }};
+create keyspace ycsb WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor': {{replication_factor}} };
+USE ycsb;
+create table usertable (
```

Comment:
Seems like table schema is hard-coded using this script, while ycsb package should be able to create tables with different number of fields: https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/e75b755b70b93a96e507a410447e7a4e59ff4dfe/perfkitbenchmarker/linux_packages/ycsb.py#L476